compiler.rb

```diff
@@ -81,7 +81,6 @@
 pp api if ENV['COMPILER_DEBUG']
 
 config = Provider::Config.parse(File.join(catalog, provider), api)
-config.validate
```

Comment:
`Config.parse` above already calls the `validate` method. This caused the validation to happen twice.

Reply:
Did that cause a side effect? `validate` should be idempotent.

Reply:
This should not be an issue with a new proposal. Let's sync offline first.

Reply:
As we agreed let's move this change to a future (separate) PR.